task_management_app/views.py

```diff
@@ -0,0 +1,295 @@
+from django.shortcuts import render,redirect
+from django.views import View
+from django.shortcuts import render, redirect, get_object_or_404
+from django.http import HttpResponse
+from django.contrib.auth import authenticate, login,logout
+from .models import Task, User, Comment
+from .models import User
+from django.contrib import messages
+from django.views import View
+from django.contrib.auth import get_user_model
+from django.db.models import Q
+from django.dispatch import receiver
+from django.core.mail import send_mail
+from django.conf import settings
+from django.http import HttpResponse
+import datetime
+
+# Create your views here.
+User = get_user_model()
+class loginform(View):
+    def get(self, request):
+        return render(request, 'login.html')
+
+    def post(self, request):
+        email = request.POST.get('email')
+        password = request.POST.get('Password')
+
+        if not email or not password:
+            messages.error(request, "Email aur password zaruri hai.")
+            return redirect('loginform')
+
+        user = authenticate(request, email=email, password=password)
+
+        if user is not None:
+            login(request, user)
+            return redirect('homepage') 
+        else:
+            messages.error(request, "Authentication failed. Please check your credentials.")
+            return redirect("loginform")
+
+
+class signupform(View):
+  def get(self,request):
+    return render(request,'registration.html')
+  def post(self,request):
```

Comment:
Please use proper spacing and add docstring. (Follow PEP8)

Reply:
and reformat code via Black